service/src/main/java/com/codeforcommunity/processor/ProtectedSiteProcessorImpl.java

```diff
@@ -1,27 +1,18 @@
 package com.codeforcommunity.processor;
 
-import static org.jooq.generated.Tables.ADOPTED_SITES;
-import static org.jooq.generated.Tables.SITES;
-import static org.jooq.generated.Tables.SITE_ENTRIES;
-import static org.jooq.generated.Tables.STEWARDSHIP;
+import static org.jooq.generated.Tables.*;
```

Comment:
More * imports. ❓ Are these being generated automatically when you run mvn spotless:apply?